### PR TITLE
fix: aws/eks module version

### PR DESCRIPTION
**Pull request by diabhey**
#13 
 - eks module hardcoded to non-breaking version v17.24.0

### CODE REVIEW

File: aws/eks/main.tf [hunk 1]
```diff
@@ -74,17 +76,14 @@ data "aws_eks_cluster_auth" "cluster" {
...
-}
 
 module "vpc" {
   source  = "terraform-aws-modules/vpc/aws"
   version = "~> 3.0"
 
   name                 = var.cluster_name
   cidr                 = "10.0.0.0/16"
-  azs                  = data.aws_availability_zones.available.names
+  azs                  = ["eu-central-1a", "eu-central-1b", "eu-central-1c"]
```
Comment (cocheok) [inline aws/eks/main.tf:86]: did we add some hardcode here?

File: aws/eks/main.tf [hunk 2]
```diff
@@ -4,6 +4,8 @@
 
 module "eks" {
   source          = "terraform-aws-modules/eks/aws"
+  # Do not upgrade to v18.0. and above as it will break the changes
+  version = "v17.24.0"
```
Comment (cocheok) [inline aws/eks/main.tf:8]: here something like var.version

File: aws/eks/main.tf (hunk 1 above)
Comment (diabhey) [reply aws/eks/main.tf:86]: this will be removed

File: aws/eks/main.tf (hunk 2 above)
Comment (diabhey) [reply aws/eks/main.tf:8]: you can't use variable for the version

**Pull request merged**